test/e2e/conftest.py

```diff
@@ -0,0 +1,120 @@
+from dataclasses import dataclass
+import subprocess
+import pytest
+import kubernetes
+from pathlib import Path
+
+from helpers import save_akri_logs
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--distribution", action="store", help="Specify distribution to use"
+    )
+    parser.addoption("--test-version", action="store", help="version to test")
+    parser.addoption(
+        "--local-tag", action="store", default="pr-amd64", help="tag for local images"
+    )
+    parser.addoption("--use-local", action="store_true", help="use local images if set")
+    parser.addoption("--release", action="store_true", help="use released helm chart")
+
+
+@dataclass
+class Distribution:
+    name: str
+    kubeconfig: Path
+    kubectl: str
+
+
+@pytest.fixture(scope="session")
+def distribution_config(pytestconfig):
+    distribution = pytestconfig.getoption("--distribution", None)
+    if distribution == "k3s":
+        yield Distribution("k3s", Path.home() / ".kube/config", "kubectl")
```

Comment:
Should we update the tests and workflow to match the distribution defaults to better enable running the tests locally without having to copy over the kubeconfig?
```suggestion
        yield Distribution("k3s", Path.home() / "/etc/rancher/k3s/k3s.yaml", "kubectl")
```

Reply:
I'm not sure of the added benefit for local runs, as I think most people will either have a long running instance they use for tests (so probably already configured) or use tools like k3d or rancher desktop that already manage the configuration.

In fact I first wanted to just get rid of that element here (and let the python kubernetes client handle it the same way the `helm` or `kubectl` commands do), but wasn't really sure it was the right thing to do.